src/osd/PGLog.h

```diff
@@ -440,6 +440,9 @@ struct PGLog : DoutPrefixProvider {
 	assert(get_can_rollback_to() == head);
       }
 
+      // make sure our buffers don't pin bigger buffers
+      e.mod_desc.trim_bl();
```

Comment:
ah, this used to be here but was accidentally removed in 5e0ec06376f832d32a6b1af390f925a59b03798f